experimental/C/src/AnytimePrime.lf

```diff
@@ -0,0 +1,67 @@
+/**
+ * This program is a concept demonstration showing how the check_deadline()
+ * function works. This program performs "anytime computation" (https://en.wikipedia.org/wiki/Anytime_algorithm)
+ * to find the largest prime within the given deadline. The check_deadline()
+ * function is called after checking each number if it is prime. If the
+ * check_deadline() is called after the deadline has passed, the function calls
+ * the deadline handler and returns true. Then the program outputs the largest
+ * prime found by that time and exits.
+ *
+ * For more discussion on check_deadline(), see: https://github.com/lf-lang/lingua-franca/issues/403
+ *
+ * @author Hokeun Kim (hokeunkim@berkeley.edu)
+ * @author Edward A. Lee (eal@berkeley.edu)
+ * @author Marten Lohstroh (marten@berkeley.edu)
+ */
+target C {
+    fast: true,
+    files: ["/lib/c/reactor-c/core/utils/vector.h",
+            "/lib/c/reactor-c/core/utils/vector.c"]
+};
+
+preamble {=
+#include "vector.c"
+=}
+
+reactor Prime {
+    output out:{=long long=};
+    reaction(startup) -> out {=
+    int num_primes = 1;
```

Comment:
Indentation is odd. Should use spaces, not tabs.

Reply:
Thanks for catching!